distribution/ddf-common/src/main/resources/bin/ddfsolr

```diff
@@ -72,16 +92,15 @@ print_messages() {
     fi
 }
 
-
-if [ "$COMMAND" = "stop" ]; then
-    $SOLR_EXEC stop -p $SOLR_PORT
+if [ "$COMMAND" == "stop" ]; then
+	stop_solr
 else
     # Read protocol from properties file
     SOLR_PROTOCOL=$($GET "solr.http.protocol")
 
     # Do not proceed if required properties are missing
     if [ -z "$SOLR_PROTOCOL" ]; then
-        echo "Cannot determine Solr protocol (http or https) from custom.system.properties file. Exiting."
```

Comment:
Looks like `Exiting` accidentally got changed.

Reply:
It's the new way to terminate a batch file. You don't exit it, you exii it.